recipes/marisa-trie/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{%set name = "marisa-trie" %}
```

Comment:
Good question. I'm sort of against because
a) this library directly wraps the C++ library code
b) it's `marisa-trie` on PyPi

That said, given that `conda` can support C++ builds I'm not sure that b) is avery strong point.